src/main/java/com/training/assignments/Assignment.java

```diff
@@ -0,0 +1,79 @@
+package com.training.assignments;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Assignment {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    /**
+     * dataConversion from one type to another
+     */
+    public void dataConversion(){
+        //int to double and vice-versa
+        int number = 55;
+        double doubleNumber=number;
+        logger.info("converion of int to double-->"+doubleNumber);
+        int num=(int)doubleNumber;
+        logger.info("converion of double to int-->"+num);
+        //double to float and vice-versa
+        float floatNumber=(float)doubleNumber;
+        logger.info("converion of double to float-->"+floatNumber);
+        double dNumber=floatNumber;
+        logger.info("converion of float to double-->"+dNumber);
+        //long to byte and vice-versa
+        long longNum = 11005987L;
+        byte byteNum= (byte)longNum;
+        logger.info("converion of long to byte = "+byteNum);
+        long lNum=byteNum;
+        logger.info("converion of byte to long = "+lNum);
```

Comment:
Done create lNum variable here,  use the longNum variable and print longNum variable in log